dev/integration_tests/ios_platform_view_tests/README.md

```diff
@@ -0,0 +1,13 @@
+# ios_platform_view_test
+
+A simple app contains:
+
+* A home with with a button that pushes a new page into the scene.
+* A page contains a platform view, a button and a text.
+    * Press the button will update the text.
+
+We use this app to test platform views in general such as platform view creation, destruction and thread merging(iOS only).
+
+# e2e tests
+
+We can possibly move the e2e tests to the flutter/engine repo when e2e is running on engine ci.
```

Comment:
nit: let's track this in an issue instead of here

Reply:
done

Reply:
Added it to the same issue I created to track enabling framework dependency in engine repo for testing. https://github.com/flutter/flutter/issues/51892